test/parallel/test-https-connect-address-family.js

```diff
@@ -36,8 +36,13 @@ function runTest() {
 }
 
 dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
-  if (err)
+  if (err) {
+    if (err.code === 'ENOTFOUND') {
+      common.skip('localhost does not resolve to ::1');
```

Comment:
@thefourtheye It's not really flaky. It's fail-every-time. The only reason CI is green is because of the setting in `parallel.status` that basically says "if this test fails, mark it as passing anyway" which is what I'm trying to get rid of in this change.

Example: https://ci.nodejs.org/job/node-test-commit-linux/4243/nodes=centos5-32/console

Note that it's green despite the existence of this:

```
not ok 560 parallel/test-https-connect-address-family # TODO : Fix flaky test
# /home/iojs/build/workspace/node-test-commit-linux/nodes/centos5-32/test/parallel/test-https-connect-address-family.js:40
#     throw err;
#     ^
# 
# Error: getaddrinfo ENOTFOUND localhost
#     at errnoException (dns.js:28:10)
#     at GetAddrInfoReqWrap.onlookupall [as oncomplete] (dns.js:92:26)
  ---
  duration_ms: 0.234
```